components/sup/src/command/shell.rs

```diff
@@ -42,7 +42,7 @@ pub fn sh() -> Result<()> {
 
 fn set_path() -> Result<()> {
     let mut paths = String::new();
-    for path in try!(busybox_paths()).iter() {
+    for path in try!(interpreter_paths()).iter() {
         if !paths.is_empty() {
             paths.push(':');
```

Comment:
This is a platform specific path separator and will break if multiple interpreter paths are specified on windows.  We moved to using `env::join_paths()` in https://github.com/habitat-sh/habitat/pull/1483/files#diff-b9b27ac18067cb62ebad852e9abc0b10R192 

Since this seems to be a pattern in a few places, maybe we should offer a function for this in core::os::env?